spec/integ/kicking.spec.js

```diff
@@ -176,6 +176,118 @@ describe("Kicking", () => {
 });
 
 
+describe("Banning", () => {
```

Comment:
file says kicking
test says banning

new test file time?

Reply:
How about renaming `kicking.spec.js` to `kicking-banning.spec.js`? I think they go well in one file.